### ISSUE TITLE
Serve page under multiple links

### ISSUE THREAD

**Issue by Zibelas** [ocpsoft/rewrite#292]
I'm facing a small problem with linking the same page under multiple links and I'm stuck on the last step. 

- I have a page physically under localhost:8080/home/index.xhtml
- I added `                .addRule(Join.path("/home/en/{path}.xhtml").to("/home/{path}.xhtml"))
                .addRule(Join.path("/home/de/{path}.xhtml").to("/home/{path}.xhtml"))`
- The page can now be reached on all 3 links.
- I have a command link that routes me to home/en/index?faces-redirect=true (or de, depending on current language)
- if the current url was home/index.xhtml it gets updated to home/de/index.xhtml
- my problem is now, that only the last rule gets executed, the url doesn't change over my command link. If I enter it directly into the browser, it works. The url stays as well if I navigate to different pages of the app but as soon as I click the button, it changes to whatever I have as last rule under the ConfigurationProvider

I only want to display the url correctly, all i18n, routing/ navigation is already working. I only have the configuration manager. What am I missing?

**Comment by tandraschko** [comment]
@Zibelas i currently have the same problem :D did you find a solution?

**Comment by Zibelas** [comment]
yes, it is working for me now. I exchanged <h:commandLink /> to a normal <a href/>. The url is updating now correctly, the page can be reached under both addresses without duplicating the source page.
